config/dev.exs

```diff
@@ -55,12 +55,12 @@ config :oli, OliWeb.Endpoint,
     host: System.get_env("HOST", "localhost"),
     port: String.to_integer(System.get_env("PORT", "443"))
   ],
-  https: [
-    port: String.to_integer(System.get_env("HTTPS_PORT", "443")),
-    otp_app: :oli,
-    keyfile: System.get_env("SSL_KEY_PATH", "priv/ssl/localhost.key"),
-    certfile: System.get_env("SSL_CERT_PATH", "priv/ssl/localhost.crt")
-  ],
+  # https: [
```

Comment:
is leaving this commented intentional? If so maybe we should just remove it altogether

Reply:
I think https is necessary in the dev environment due to CSRF issues, this config should guarantee that. I'm wondering if removing this might cause other issues

Reply:
urgh. Nope. Removed it.

Reply:
Commented it out just for local purpose